src/Microsoft.Identity.Web/TokenCacheProviders/MsalAbstractTokenCacheProvider.cs

```diff
@@ -121,16 +67,11 @@ protected virtual Task OnBeforeWriteAsync(TokenCacheNotificationArgs args)
         /// <summary>
         /// Clear the cache.
         /// </summary>
-        /// <returns>A <see cref="Task"/> that represents a completed clear operation.</returns>
-        public async Task ClearAsync()
+        /// <param name="homeAccountId">HomeAccountId for a user account in the cache.</param>
+        public async Task ClearAsync(string homeAccountId)
```

Comment:
Perhaps we can rename this method or specify in the summary that this clears user token cache? To make the purpose of this method clearer.